test/pagila.sh

```diff
@@ -5,7 +5,7 @@ set -e
 die() { echo "$*" >&2 ; exit 1; }
 
 test -z "$CID" && die "Please specify \$CID variable"
-test -d common || die "Please run me from git root directory"
+# test -d common || die "Please run me from git root directory"
```

Comment:
Is this needed just so that we do not download pagila multiple times?

Reply:
yes